.github/workflows/tests.yml

```diff
@@ -172,8 +172,23 @@ jobs:
       - name: Test
         run: cross test --no-fail-fast --target=${{ matrix.target }} --features=std
 
-  macos-link:
-    name: macOS ARM64 Build/Link
+  macos-link-arm64:
+    name: macOS ARM64 Build/Link (arm64 host)
```

Comment:
I think it would be fine to only do the `aarch64` build/link on the `macos-14` host. I don't think we need two jobs.